docs/resources/node_actions.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: "cloudamqp"
+page_title: "CloudAMQP: cloudamqp_node_actions"
+description: |-
+  Invoke actions on a specific node (e.g. restart RabbitMQ).
+---
+
+# cloudamqp_node_actions
+
+This resource allows you to invoke actions on a specific node.
+
+Only available for dedicated subscription plans.
+
+## Example Usage
+
+Already know the node identifier (e.g. from state file)
+
+```hcl
+# New recipient to receieve notifications
+resource "cloudamqp_node_actions" "node_action" {
+  instance_id = cloudamqp_instance.instance.id
+  node_id = <node_id>
+  action = "restart"
+}
+```
+
+Using data source `cloudamqp_nodes` to restart RabbitMQ on all nodes
+
+```hcl
+data "cloudamqp_nodes" "list_nodes" {
+  instance_id = cloudamqp_instance.instance.id
+}
+
+resource "cloudamqp_node_actions" "node_action" {
+  instance_id = cloudamqp_instance.instance.id
+  action = "restart"
+  count = length(data.cloudamqp_nodes.list_nodes.nodes)
+  node_id = data.cloudamqp_nodes.list_nodes.nodes[count.index].node_id
+}
```

Comment:
```suggestion
resource "cloudamqp_node_actions" "restart_01" {
  instance_id = cloudamqp_instance.instance.id
  action = "restart"
  node_id = data.cloudamqp_nodes.list_nodes.nodes[0].node_id
}

resource "cloudamqp_node_actions" "restart_02" {
  instance_id = cloudamqp_instance.instance.id
  action = "restart"
  node_id = data.cloudamqp_nodes.list_nodes.nodes[1].node_id
  depends_on = [
    cloudamqp_node_actions.restart_01,
  ]
}

resource "cloudamqp_node_actions" "restart_03" {
  instance_id = cloudamqp_instance.instance.id
  action = "restart"
  node_id = data.cloudamqp_nodes.list_nodes.nodes[2].node_id
  depends_on = [
    cloudamqp_node_actions.restart_01,
    cloudamqp_node_actions.restart_02,
  ]
}
}
```

Reply:
Chain node action resources to invoke action on one node at the time. Each new invoke waits until the previous is finished to avoid running them in parallell.

Reply:
Added this in f12de7a